packages/cms-lib/functions.js

```diff
@@ -7,14 +7,34 @@ const isObject = require('./lib/isObject');
 const { getCwd } = require('./path');
 
 const functionBody = `
-exports.main = ({ accountId, body, params }, sendResponse) => {
-  console.log('Your HubSpot account ID: %i', accountId);
-  sendResponse({
-    statusCode: 200,
-    body: {
-      message: 'Hello, world!',
-    },
-  });
+// Require axios library to make API requests
```

Comment:
Do we purposefully have a line break as the first line?

Reply:
Not really. Addressed via 119396c6f1c267d6c954465c26d643b47010f710